jenkins/ci.suse.de/cloud-view.yml

```
@@ -0,0 +1,324 @@
+- view:
+    name: Cloud/1) Cloud Gating/Gating Overview
+    view-type: list
+    jobNames:
+      - cloud-8-gating
+      - cloud-9-gating
+      - cloud-mkcloud7-gating
+      - cloud-mkcloud8-gating
+      - cloud-mkcloud9-gating
+    columns: &default_columns
+      - StatusColumn
+      - WeatherColumn
+      - JobColumn
+      - LastSuccessColumn
+      - LastFailureColumn
+      - LastDurationColumn
+      - BuildButtonColumn
+- view:
+    name: Cloud 9 Gating Tests
+    view-type: list
+    regex: (cloud-(ardana|crowbar)9-job-gate-.*|cloud-mkcloud9-job-(ha|2nodes|4nodes-linuxbridge))-x86_64$
+    columns:
```

```suggestion
    columns: *default_columns
```
Maybe. Then delete the next line.

---

Thanks for hint !